### ISSUE TITLE
Orderlist: Documentation about control buttons is incorrect

### ISSUE THREAD

**Issue by ribrown-bishs** [primefaces/primereact#4509]
### Describe the bug

On the page https://primereact.org/orderlist/#accessibility it describes some functionality that isn't there. 
> Controls buttons are button elements with an aria-label that refers to the aria.moveTop, aria.moveUp, aria.moveDown and aria.moveBottom properties of the [locale](https://primereact.org/locale/) API by default, alternatively you may usemoveTopButtonProps, moveUpButtonProps, moveDownButtonProps and moveBottomButtonProps to customize the buttons like overriding the default aria-label attributes.

The labels aren't there (in the demo on this page) and these are also missing on the API page. 

### PrimeReact version

9.4.0

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Expected behavior

This shouldn't be in the documentation (yet? 😏 ) as the functionality should match the documentation.

**Comment by melloware** [comment]
PR Submitted
